package.json

```diff
@@ -1,8 +1,13 @@
 {
-  "name": "morpho-blue",
+  "name": "@morpho-org/morpho-blue",
   "description": "Morpho Blue Protocol",
   "license": "BUSL-1.1",
-  "version": "1.0.0",
+  "version": "v1.0.0",
+  "files": [
+    "src",
+    "README.md",
+    "LICENSE"
+  ],
```

Comment:
Can we clarify why `lib` is included in the package bundle before merging this pls?

Reply:
A bit weird since its working well on this repo..

![image](https://github.com/morpho-org/morpho-blue/assets/61523188/7da53c15-ed7f-4ce1-a698-968151c466cd)

Reply:
You don't have to publish a package to npm to know this ahah: `yarn pack`

Reply:
Alright I confirm so this discussion can be resolved

Reply:
Wanted to reproduce the exact same flow (idk if publish has some extra things that pack has not), btw Ive just deleted the package 😆